app/components/badge/status-badge.js

```diff
@@ -0,0 +1,33 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+  _statusStyle: Ember.computed('status', 'size', 'float', function() {
+    let size = this.get('size');
+    let fontsize = size*0.7;
+    let pad = (size-fontsize)*0.5;
+    let width = 7*size;
+    let float = 'float: none';
+    if(this.get('float')){
+      float = 'float: ' + this.get('float') + '; margin-top:-20px';
+    }
+    switch (this.get('status')) {
+      case `init`: return `background-color: #cc5490; width:${width}px; font-size:${fontsize}px; padding:${pad}px; ${float}`.htmlSafe();
```

Comment:
I think this should be done using sass and classes. You can make it dynamic as well, so have a `status-badge-${status}`, and the sass defines classes for `status-badge-init`, `status-badge-spec-design`, etc.